.travis.yml

```diff
@@ -37,20 +35,42 @@ install: "echo 'Skipping install stage, dependencies will be downloaded during b
 # 1. Install & Unit Test APIs
 # 2. Assemble DSpace
 script:
+  # 0. run mvn help:effective-settings to confirm credentials are in use
+  - "mvn help:effective-settings --settings ./settings.xml"
   # 1. [Install & Unit Test] Check source code licenses and run source code Unit Tests
   #    (This explicitly skips building the 'dspace' assembly module, since we only want to do that ONCE.)
   #        license:check => Validate all source code license headers
   #        -Dmaven.test.skip=false => Enable DSpace Unit Tests
   #        -P !dspace => SKIP full DSpace assembly (will do below)
   #        -B => Maven batch/non-interactive mode (recommended for CI)
   #        -V => Display Maven version info before build
-  - "mvn clean install license:check -Dmaven.test.skip=false -P !dspace -B -V"
+  - "mvn clean install license:check -Dmaven.test.skip=false -P !dspace -B -V --settings ./settings.xml"
   # 2. [Assemble DSpace] Ensure assembly process works (from [src]/dspace/), including Mirage 2
   #        -Dmirage2.on=true => Build Mirage2
   #        -Dmirage2.deps.included=false => Don't include Mirage2 build dependencies (We installed them in before_install)
   #        -P !assembly => SKIP the actual building of [src]/dspace/dspace-installer (as it can be memory intensive)
-  - "cd dspace && mvn package -Dmirage2.on=true -Dmirage2.deps.included=false -P !assembly -B -V"
+  - "mvn package -Dmirage2.on=true -Dmirage2.deps.included=false -P !assembly -B -V --settings ./settings.xml"
+
 notifications:
   hipchat:
     rooms:
       secure: IJ/e7tbe7xZeC3BAMrQUgKOe9ZoayqhQRLgeCONw1KlcW7Vncq3SXVHyP5p5kssPEgt+7rbRbtoFqwa3seoIxXdW2E4NpOoo6TNYsrJLL1GC3GQAZ6Tqawbw+vnbsjRpBkDFNV9cQQFZaVBkQzRMO6BHVQmOiJ8dLJMxLbmFrgk=
+env:
+  global:
+  # Give Maven 1GB of memory to work with
+  - MAVEN_OPTS=-Xmx1024M
+
+# ENCRYPTED TOKENS: one set per developer (all other tokens will be un-decryptable and thus ignored)
+  # use these encrypted tokens for access to atmire's private Maven repository
+
+    # UMLSO/DSpace
+  - secure: exONiATp9HEvocvWhlpYN5Yb3UkemjvS6rjXGiXkTMF3Oq099OTamXUpEXYKfRC4+aNeUOHOSKQrHSvBfR3lS4khqfXv53OTup2bGYcAS0R5KatHslU89CfD+p5JsT2ki/lQ1nBmC+V1BU1UbuspxzwKRM00QNXQAF7tfi6gmf8=
+  - secure: aM9RQQDTgdQWZat3pV8vrgALHEi1eNNkaP8ybeZojQ7zE4EdHBkJI7Gf+e2zsz/f8Z1+CxMxePsY0VXDB7pyF8TxXNUPZgBEaFtsP1l7l2m0lxzLQZ45XEeJVPGFaivIf/h2ySZoimv0MoemGxeaKjG0/Il9l/QgUGG5Dfod7rs=
+
+    # hardyoyo/DSpace
+  - secure: G3ydWVr/cW2uEbqUFtcRF63P1+rJX31xL5xgLsfL3Y6lxbHk4aWhDVi/rRD4S+BfltZ6RYnEMP1CDZIEyJVV+bea8UhW+5fAR4uJn4GmZyIn80si96SBhRM0QbSzvyXofwmM1tTdELSo7xeSKgTsOz3/g3jyIo3BT5naS1kalKI=
+  - secure: b3TFpfhCANOppolFhEfh7NOJjlf8v/OWUdRzEeBpyh1QEHlWJcwdrRqsDpVX+j3Ttd6eqnyCozNytyM4+D7zX9UbUERwmJjKmrpudcY22MiodhKsffqskMbPDp/wx4/92B/wxeDx4VKQe0ocho/EOElxJ/iG/og/C+o1iECCRLs=
+
+   # atmire/DSpace
```

Comment:
This may be helpful in certain instances, however, when making a PR, you are actually relying on your own credentials as far as Travis-CI is concerned, so you'll need to encrypt these credentials using your own repository's key. For example, you probably typed the following for the atmire/DSpace lines above: `travis encrypt -r atmire/DSpace "export CI_ATMIRE_USERNAME=obfuscated"`however, in order for Travis-CI to sucessfully build this PR, you'll need to copy/paste the results of this command:  `travis encrypt -r KevinVdV/DSpace "export CI_ATMIRE_USERNAME=obfuscated"` into the .travis.yml file, in this same section.